[16.0 ] purchase_order_product_recommendation:  Name 'abs' is not defined

<!-- Provide a general summary of the issue in the Title above.
Before opening the issue, please check that there isn't another issue addressing the same problem already created -->

## Module
purchase_order_product_recommendation
##Error
UncaughtPromiseError > OwlError
Promessa não capturada > An error occured in the owl lifecycle (see this Error's "cause" property)
OwlError: An error occured in the owl lifecycle (see this Error's "cause" property)
    OwlError@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1240:1
    handleError@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1472:101
    handleError@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:2099:29
    _render@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1497:19
    render@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1495:6
    initiateRender@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1563:47

Caused by: EvaluationError: Name 'abs' is not defined
    EvaluationError@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1245:430
    _evaluate@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1281:232
    _evaluate@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1285:32
    evaluate@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1294:8
    applyBinaryOp@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1265:95
    _evaluate@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1282:102
    _evaluate@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1282:212
    evaluate@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1294:8
    evaluateExpr@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:1144:109
    getRowClass/classNames<@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:4638:103
    getRowClass@https://192.168.20.100/web/assets/2644-a447375/web.assets_backend.min.js:4638:70
    template@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js line 2057 > Function:23:21
    callTemplate@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1718:129
    template@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js line 2057 > Function:35:32
    callTemplate@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1718:129
    template@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js line 2057 > Function:163:11
    _render@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1496:96
    render@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1495:6
    initiateRender@https://192.168.20.100/web/assets/2643-c395e48/web.assets_common.min.js:1563:47

## To Reproduce
ver 16.0

## Describe the bug
File: /wizards/purchase_order_recommendation_view.xml
Line 41: decoration-danger="units_virtual_available &lt; 0 and units_included &lt; abs(units_virtual_available)"

Change to:
decoration-danger="units_virtual_available &lt; 0 and units_included &lt; 'abs(units_virtual_available)'"

Work for me

--- a/purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml
+++ b/purchase_order_product_recommendation/wizards/purchase_order_recommendation_view.xml
@@ -38,7 +38,7 @@
                             <tree
                                 decoration-info="purchase_line_id != False"
                                 decoration-success="units_included &gt; 0 and not purchase_line_id"
-                                decoration-danger="units_virtual_available &lt; 0 and units_included &lt; abs(units_virtual_available)"
+                                decoration-danger="units_virtual_available &lt; 0 and units_included &lt; units_virtual_available * (1 if units_virtual_available &gt; 0 else -1)"
                                 decoration-muted="units_received == 0 and units_delivered == 0"
                                 create="0"
                                 delete="0"

